src/main/java/com/epam/lab/paymentsystem/service/impl/UserServiceImpl.java

```diff
@@ -7,10 +7,15 @@
 import com.epam.lab.paymentsystem.entities.enums.Roles;
 import com.epam.lab.paymentsystem.exception.LoginAlreadyExistsException;
 import com.epam.lab.paymentsystem.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
 
+@Service
 public class UserServiceImpl implements UserService {
 
+    @Autowired
     private final UserDAO userDAO;
```

Comment:
`final` is redundant here, also sometimes autowiring might not work due to `final` (like in the case of autowiring through setter)